src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Helpers.kt

```diff
@@ -159,4 +161,22 @@ private val prettyTime = PrettyTime()
 fun getFriendlyRelativeDateTime(date: Date) : String
 {
     return prettyTime.format(date)
+}
+
+fun jsonToStringMap(json: String): Map<String, String>
+{
+    val element = JsonParser().parse(json)
+    val type = TypeToken.getParameterized(Map::class.java, String::class.java, String::class.java).type
+    return Serializer.instance.gson.fromJson<Map<String, String>>(element, type)
+}
+
+/**
+ * This implementation is to support re-usability
+ * hence, the reason for generics.
+ */
+fun <T: Any> jsonStringToGenericList(json: String, clazz: Class<T>): List<T>
```

Comment:
For consistency with previous function please use either:
- `jsonToStringMap` and `jsonToGenericList`
- `jsonStringToStringMap` and `jsonStringToGenericList`

Reply:
I am not sure this is consistent. The first one returns a Map and the sent a List.

Reply:
It's about naming - just suggesting a common prefix for both functions: either `jsonTo...` or `jsonStringTo...`